drivers/bluetooth/hci/userchan.c

```diff
@@ -229,7 +229,7 @@ static void rx_thread(void *p1, void *p2, void *p3)
 				LOG_ERR("Not enough space in buffer %zu/%zu",
 					buf_add_len, buf_tailroom);
 				net_buf_unref(buf);
-				continue;
+				break;
```

Comment:
For both cases, would it make sense to do 
```c

			len -= decoded_len;
			frame_start += decoded_len;
```

and still `continue`? Even if we do not have enough memory for this packet, we may have for the next, right?